doc/tutorials/language-bindings.md

```diff
@@ -15,6 +15,89 @@ In this section, we will explain the how to write a language binding for Elektra
    4. other programming language integrations (streams, hash-codes, identity, ...)
    5. returned errors from kdb functions (what this issue here is about)
 
+## CMake Integration
+
+### Building
+
+After setting up our project, we have to add the projects directory to `src/bindings/CMakeLists.txt`.
+
+```cmake
+check_binding_included ("our_bindings" IS_INCLUDED)
+if (IS_INCLUDED)
+	add_subdirectory (our_bindings_directory)
+endif ()
+```
+
+At first we want to make sure that the build tools and compilers we need for the project are installed. We can use `find_program (BUILD_TOOL_EXECUTABLE build_tool)` to find our `build_tool` program. The result of the search will be stored in `BUILD_TOOL_EXECUTABLE`, so now we can use an if block to include the bindings in the build, if the program exists or exclude it, if it doesn't.
+
+```cmake
+if (CARGO_EXECUTABLE)
+    add_binding (our_bindings)
```

Comment:
Formatting inconsistent (here are spaces). And describe add_binding and link to cmake/Modules/LibAddBinding.cmake for further information.